mw-webapp/src/globalStore/ThemeStore.ts

```diff
@@ -95,6 +95,10 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     [Theme.DARK]: "#FFFFFF3E",
     [Theme.LIGHT]: "#784DEA3E",
   },
+  dropdownStrokeColor: {
+    [Theme.DARK]: "#FFFFFF3E",
```

Comment:
On the maket there are different value for this token

Reply:
fixed